### PR TITLE
Azure Government LUIS region added

### PR DESCRIPTION
**Pull request by fgarofalo56**
Fixes #

## Proposed Changes

Updated mapping to include the new GA release for LUIS in Azure Government. Maps to the virginia endpoint
Updates to MSBot utils.ts file to include the mappings for LUIS in Azure US Gov.
Update to MSBot-conect-luis.ts to for Azure US Gove region to us virginia when users pass usgovirginia to match the endpoint

## Testing
-ran build local with no errors.

### CODE REVIEW

**Comment by fgarofalo56** [comment]
How do i run the travis tests locally?  When i run "npm run build" I do not get any errors.  I really need the new endpoint infor added for Azure Gov as it went GA last week.

File: packages/MSBot/src/msbot-connect-luis.ts [hunk 1]
```diff
@@ -97,6 +97,10 @@ async function processConnectLuisArgs(config: BotConfiguration): Promise<BotConf
         throw new Error('bad or missing --authoringKey');
     }
 
+    if (!args.region || args.region === 'usgovvirginia') {
```
[the remark this answers is not in the document]
Comment (cleemullins) [reply packages/MSBot/src/msbot-connect-luis.ts]: Agreed. That's probably not the right behavior.

**Comment by cleemullins** [comment]
@fgarofalo56 Are you able to get the merge conflicts resolved here? The build issues are all fixed now.

**Comment by fgarofalo56** [comment]
I am.  Just let me know what you need me to do and I will.

Thanks,

Frank

File: packages/MSBot/src/msbot-connect-luis.ts (hunk 1 above)
[the remark this answers is not in the document]
Comment (fgarofalo56) [reply packages/MSBot/src/msbot-connect-luis.ts]: My intent for this was if the user typed/used 'usgovvirginia' the endpoint defaults to 'virginia' like it does in the Gov LUIS portal.  If this not the right behavior I can strip it out.

**Comment by cleemullins** [comment]
The build error is:
```
lerna notice cli v3.2.1
lerna info versioning independent
lerna info Bootstrapping 7 packages
lerna info Installing external dependencies
lerna ERR! npm ci exited 1 in 'botdispatch'
lerna ERR! npm ci stderr:
npm ERR! Unexpected token < in JSON at position 27
npm ERR! A complete log of this run can be found in:
npm ERR!     /home/travis/.npm/_logs/2018-11-14T21_03_49_848Z-debug.log
lerna ERR! npm ci exited 1 in 'botdispatch'
```

**Comment by fgarofalo56** [comment]
Not sure what is needed to make this build correctly, as it is erroring on botdispatch which I did not change or do anything with.  Closeting the pull request.

**Pull request closed without merging**